filebeat/input/filestream/fswatch_test.go

```diff
@@ -856,17 +855,21 @@ func BenchmarkGetFilesWithFingerprint(b *testing.B) {
 		err := os.WriteFile(filename, []byte(strings.Repeat(content, 1024)), 0777)
 		require.NoError(b, err)
 	}
-	paths := []string{filepath.Join(dir, "*.log")}
-	cfg := fileScannerConfig{
+
+	fsConfig := fileScannerConfig{
 		Fingerprint: fingerprintConfig{
 			Enabled: true,
 			Offset:  0,
 			Length:  1024,
 		},
 	}
-	s, err := newFileScanner(paths, cfg)
+
+	paths := []string{filepath.Join(dir, "*.log")}
+	s, err := newFileScanner(paths, fsConfig)
 	require.NoError(b, err)
 
+	b.ResetTimer()
```

Comment:
Resetting the timer as operations before this are not relevant to the actual benchmark i.e., creation of dummy files, etc.

Reply:
> Did you make sure that this ResetTimer was in the base benchmark too before comparing them?

Yes.